src/vfio_device.rs

```diff
@@ -513,17 +569,23 @@ impl VfioDeviceInfo {
 pub struct VfioDevice {
     device: File,
     flags: u32,
-    group: VfioGroup,
     regions: Vec<VfioRegion>,
     irqs: HashMap<u32, VfioIrq>,
-    mem: GuestMemoryMmap,
+    #[allow(dead_code)]
+    group: VfioGroup,   // Take owner of the group group object, otherwise it will be dropped.
```

Comment:
Would you mind explaining why this could be `dead_code`?

Reply:
It's a temporary code and will be removed by later commits.
The VfioDevice object needs to hold an reference to the associated VfioGroup object, otherwise the
VfioGroup object will be destructed.